client/my-sites/migrate/section-migrate.jsx

```diff
@@ -110,6 +110,12 @@ class SectionMigrate extends Component {
 	finishMigration = () => {
 		const { targetSiteId, targetSiteSlug } = this.props;
 
+		/**
+		 * Request another update after the migration is finishes to update the site title and information
+		 * that will come when the migration is completed.
```

Comment:
Typo `is finishes`. Does the info change during the migration, or at the end? Do we mean to say like "Request another update after the migration is finished to update the site title and other info that may have changed."?

Reply:
Thanks! That's exactly what I wanted to say. Thanks for making it sound better! Updated the comment.